### PR TITLE
Feature/graphical frequency

### PR DESCRIPTION
**Pull request by Brennan1994**
Graphical frequency partially implemented. Does not allow saving/loading yet. Is not linked into the compute.

### CODE REVIEW

File: ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs [hunk 1]
```diff
@@ -109,30 +109,30 @@ public int PeriodOfRecord
...
-        {
-            get { return _GraphicalFlows; }
-            set { _GraphicalFlows = value; NotifyPropertyChanged(); }
-        }      
+        public TableWithPlotVM MyGraphicalVM { get; set; }
 
         #endregion
         #region Constructors
+        //This supports a fresh editor
         public AnalyticalFrequencyEditorVM(ComputeComponentVM defaultCurve,  EditorActionManager actionManager) : base(defaultCurve, actionManager)
```
Comment (rnugent3) [inline ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs:122]: does it matter that this is within the analytical frequency editor?

File: ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs [hunk 2]
```diff
@@ -109,30 +109,30 @@ public int PeriodOfRecord
...
         {
             _Mean = DefaultCurveData.LP3Mean;
             _StDev = DefaultCurveData.LP3StDev;
             _Skew = DefaultCurveData.LP3Skew;
             _POR = DefaultCurveData.LP3POR;
-
+            MyGraphicalVM = new TableWithPlotVM(new GraphicalVM("Graphical Flow Frequency","Probability","Flow"), true);
```
Comment (rnugent3) [inline ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs:128]: does it matter that graphical could be stage or flow?

File: ViewModel/FrequencyRelationships/GraphicalVM.cs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        override public void LoadFromXML(XElement element)
+        {
+            EquivalentRecordLength = int.Parse(element.Attribute("EquivalentRecordLength").Value);
+            base.LoadFromXML(element);
+        }
+        private void ConfidenceLimitsAction(object arg1, EventArgs arg2)
+        {
+            GraphicalUncertainPairedData graffical = MyGraphical;
+            PairedData upperNonExceedence = graffical.SamplePairedData(.95) as PairedData;
+            PairedData lowerNonExceedence = graffical.SamplePairedData(.05) as PairedData;
```
Comment (rnugent3) [inline ViewModel/FrequencyRelationships/GraphicalVM.cs:66]: I think for consistency purposes we should use the .025 and .975

**Review by rnugent3 — APPROVED** [review]
Looks good. Thank you for prioritizing this.

File: ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs (hunk 2 above)
Comment (Brennan1994) [reply ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs:128]: It does. I'm currently not supporting stage. I want to get back into main as quickly as possible though. I've made a ticket to address this comment in a later pull request. https://github.com/HydrologicEngineeringCenter/HEC-FDA/issues/386

File: ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs (hunk 1 above)
Comment (Brennan1994) [reply ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs:122]: This is bad nomenclature. I'd like to rename this editor, as the same editor supports both graphical and analytical, despite being called analytical. It is the same in the UI. I've made a ticket for this. https://github.com/HydrologicEngineeringCenter/HEC-FDA/issues/387

File: ViewModel/FrequencyRelationships/GraphicalVM.cs (hunk 3 above)
Comment (Brennan1994) [reply ViewModel/FrequencyRelationships/GraphicalVM.cs]: This has been corrected.

**Pull request merged**